scripts/run_adversarial_live.py

```
@@ -0,0 +1,102 @@
+        env.pop("CUSTOM_BASE_URL", None)
+        env.pop("CUSTOM_API_KEY", None)
+    else:
+        env = os.environ.copy()
+        # Verify API config
+        if not env.get("CUSTOM_BASE_URL") or not env.get("CUSTOM_API_KEY"):
+            print("ERROR: CUSTOM_BASE_URL and CUSTOM_API_KEY must be set for live mode.")
+            print("Use --dry-run for offline testing.")
+            sys.exit(1)
+
+    # Run tests
+    result = subprocess.run(cmd, env=env, cwd=os.path.dirname(os.path.dirname(__file__)))
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)